Automattic/jetpack
# Story Block: Defaults to adding new media in media picker instead of replacing them

Addresses https://github.com/Automattic/jetpack/issues/19945

#### Changes proposed in this Pull Request:
This PR updates the `multiple` props used in the `MediaUpload` component so we can select multiple media by toggling each one instead of holding cmd/ctrl as it is the case at the moment. It offers a more intuitive experience for editing stories

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

* Edit a story by clicking on the pencil icon in the block toolbar
* Also try selecting the `Select Media > Media Library` option
* In both case the media library UI should default to toggling media in and out instead of replacing them
